fastlane/metadata/android/pl/full_description.txt

```diff
@@ -1,7 +1,7 @@
-<p>Normalnie użyłbyś/użyłabyś Google Translate. Problem tylko jest taki, że albo będziesz zalogowany(-a) na swoim koncie Google, albo będziesz miał(a) to samo IP co Twoje konto. Tłumaczenia mogą zostać przypięte do Twojej tożsamości. Czasami nie ma z tym problemu, innymi razy jest. Dokładnie tak, jak z przeglądaniem Wikipedii. Twórcy będą wiedzieć, jakie masz choroby, poglądy polityczne, jaką wyznajesz religię, jakiej jesteś orientacji seksualnej itd. Tłumaczenia są pod tym względem nieco gorsze.</p>
+<p>Normalnie użyłbyś Google Translate. Jednak problem polega na tym, że zostaniesz zalogowany na swoje konto Google, albo będziesz miał to samo IP co Twoje konto. Tłumaczenia mogą być powiązane z Twoją tożsamością. Czasami nie ma z tym problemu, innym razem jest. Dokładnie tak, jak z przeglądaniem Wikipedii. Twórcy będą wiedzieć, jakie masz choroby, poglądy polityczne, jaką wyznajesz religię, jakiej jesteś orientacji seksualnej itp. Tłumaczenia są pod tym względem nieco gorsze.</p>
```

Comment:
You should leave dual forms like `zalogowany(-a)` to make description more inclusive

Reply:
In the general description of something (the app here), you do not use masculine or feminine phrases like `zalogowany(-a)` or `miał(-a)` because it introduces haos. It is different in official or official letters that are already addressed to a given person. Then you should use variations such as `Pan/Pani` or `otrzymał/otrzymała` etc.

@Aga-C I can change to the infinitive 😉.

Reply:
Not exactly. We don't know the gender of the reader, that's why we are using both forms to indicate it. It becomes more and more common to use splitting to annotate both forms as equal, despite of classical grammar, where masculine form is dominant. I can see it more often in editorial guidelines to use splitting, so our texts are at the same time direct to the reader and inclusive.

Using infinitive form is ok, but we are losing the direct approach to the reader, therefore the text doesn't have as much power as the original, direct form. It has more psychological power when you say `Google spies on you` versus `Google is spying on users`.

Reply:
> We don't know the gender of the reader, that's why we are using both forms to indicate it.

We are really talking about the user or the future user of the application. Therefore it is possible to keep the masculine gender.

Similarly, in translating the program, you do not turn the male or female into words in the task, e.g. `Wyłałeś(-aś) coś tam`, but it declines for the masculine gender as a user of the program, meaning `Wysłałeś coś tam`.

He understands your indignation, but that's the way it is 😁.

Reply:
But it's done like this in applications and on web pages too (e.g. Twitter). It's becoming normal, that we consider that the person on the second side of the screen can be either male or female.
In your example, of course, we are referring to the user `użytkownik` that is masculine. And it's perfectly fine if you are saying things like `User did that`. But when referring to someone directly, you should consider that this person may like to think of herself as `użytkowniczka`. Myself I don't use that form, but it's becoming more popular, and we must adapt to how language changes, as long as these changes aren't against its rules (in terms of splitting, it's not against language rules).

In my opinion, splitting is still a more natural form than other proposed forms for gender-neutral language. It's for me the best choice, but I'll leave the decision for you. I just wanted to express my opinion, why I wrote this way.